### ISSUE TITLE
API on Publish to Apple News constantly showing errors

### ISSUE THREAD

**Issue by NapaFoodGal** [alleyinteractive/apple-news#695]
I cannot get Publish to Apple News to be compatable with my wordpress. I keep getting error messages
I have installed and uninstalled and reinstalled the plugin and it is still showing issues. Apple News cannot help me they said contact the developer.
Please advise on what to do so it works
See issues below. I am not a programer or web developer so this is greek to me. Please help

INVALID_DOCUMENT - "text" was "" should be not (null or an empty string) (keyPath components->3->components->27->text)
INVALID_DOCUMENT - "text" was "" should be not (null or an empty string) (keyPath components->3->components->55->text)
INVALID_DOCUMENT - "text" was "" should be not (null or an empty string) (keyPath components->3->components->70->text)
INVALID_DOCUMENT - "text" was "" should be not (null or an empty string) (keyPath components->3->components->100->text)
INVALID_DOCUMENT - "text" was "" should be not (null or an empty string) (keyPath components->3->components->117->text)

**Comment by svanbo** [comment]
I've encountered this problem as well, it was caused by extra "&nbsp;" characters in the post content. These appear to be added by WordPress when entering multiple empty lines while in Visual Mode (at least with the Classic editor).

A workaround is to switch to Text (HTML) mode and simply delete these characters. These are basically just space characters in a special HTML notation so they should be safe to delete. More information here:
https://www.computerhope.com/jargon/n/nbsp.htm

It would be nice if this could be fixed, since this issue seems to come up pretty often for us.
